content/en/docs/concepts/policy/resource-quotas.md

```diff
@@ -76,6 +76,10 @@ The following resource types are supported:
 | `limits.memory` | Across all pods in a non-terminal state, the sum of memory limits cannot exceed this value. |
 | `requests.cpu` | Across all pods in a non-terminal state, the sum of CPU requests cannot exceed this value. |
 | `requests.memory` | Across all pods in a non-terminal state, the sum of memory requests cannot exceed this value. |
+| `hugepages-<size>` | Across all pods in a non-terminal state, the number of
+huge page requests of the specified size cannot exceed this value. |
+| `cpu` | Same as `requests.cpu` |
+| `memory` | Same as `requests.memory` |
```

Comment:
Which should people use / is one form deprecated?

Reply:
No, both are valid. There is no hint on which one is preferred.

Reply:
This seems odd. Which one is preferred or how would readers decide which one to use?